twake/frontend/src/app/state/recoil/selectors/ThreadWrittingActivity.ts

```diff
@@ -0,0 +1,7 @@
+import { selector } from 'recoil';
+import { ChannelWrittingActivityState } from '../atoms/ChannelWritingActivity';
+
+export const ThreadWrittingActivitySelector = selector<string>({
+  key: 'ThreadWrittingActivitySelector',
+  get: ({ get }) => get(ChannelWrittingActivityState)?.threadId || '',
+});
```

Comment:
Not exactly, the goal here is to listen only to the thread, it looks like this:

```
export const ThreadWrittingActivitySelector = selectorFamily<{channelId: string, threadId: string}>({
  key: 'ThreadWrittingActivitySelector',
  get: ({threadId, channelId}) => ({ get }) => (get(ChannelWrittingActivityState(channelId)) || []).filter(status => status.threadId === threadId),
});
```

(not tested but you'll get the idea)

Reply:
Even better: 
```
export const ThreadWrittingActivitySelector = selectorFamily<{channelId: string, threadId: string}>({ key: 'ThreadWrittingActivitySelector', get: ({threadId, channelId}) => ({ get }) => (get(ChannelWrittingActivityState(channelId)) || []).filter(status => status.isWriting && status.threadId === threadId), });
```